mods/d2k/rules/structures.yaml

```diff
@@ -895,9 +895,6 @@ conyard.harkonnen:
 conyard.ordos:
 	Inherits: construction_yard
 	Buildable:
-		Queue: Building
-		BuildPaletteOrder: 1000
-		Prerequisites: ~disabled
```

Comment:
I guess this makes the construction yard appear twice with cheats on:

![image](https://cloud.githubusercontent.com/assets/756669/10716777/b4c3c88c-7b43-11e5-8df6-bcd0e2acb53d.png)